modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/dto/Activity.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.console.dto;
+
+import java.util.UUID;
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+/**
+ * Activity info.
+ */
+public class Activity extends AbstractDto {
+    /** */
+    private String grp;
+
+    /** */
+    private String act;
+
+    /** */
+    private int amount;
+
+    /**
+     * Default constructor for serialization.
+     */
+    public Activity() {
+        // No-op.
+    }
+
+    /**
+     * Full constructor.
+     *
+     * @param id Activity ID.
+     * @param grp Group.
+     * @param act Activity.
+     * @param amount Number of times activity was executed in current period..
+     */
+    public Activity(UUID id, String grp, String act, int amount) {
```

Comment:
Why Activity has Id?

Reply:
Because we save it in DB. And it extends AbstractDto.

Reply:
This is random UUID, I can find any usage of it

Reply:
This should be investigated in separate ticked.